src/utils.js

```diff
@@ -4,9 +4,9 @@ const newsApi = axios.create({
   baseURL: "https://newssite-zy4v.onrender.com/api/",
 });
 
-const fetchArticles = () => {
+const fetchArticles = (query) => {
   return newsApi
-    .get("/articles")
+    .get(`/articles${query}`)
```

Comment:
I like that you've extended the existing fetchArticles function rather than make a new one that takes a query